### DISCUSSION TITLE
[RFC] No Performer Images

### DISCUSSION THREAD

**Discussion by nimmadev** [stashapp/stash#3605] [Q&A] [answered]
## Reference Reading

i added scrappers.
reloded them
added them to identify
and run it performer were generated but with no image

**Comment by nimmadev** [comment]


**Reply by DogmaDragon** [reply]
Images for studios are not pulled at the moment. But it's being worked on <https://github.com/stashapp/stash/pull/3510>.

**Comment by DogmaDragon** [comment, marked as answer]
Go to the Performers tab > click on the Tagger > Batch Update Performers. It should update and add the images. I'm not sure why the Identify task doesn't, never used it.  
![image](https://user-images.githubusercontent.com/103123951/227842014-34d4efd3-904b-4e9b-9112-013ef684357f.png) 
![image](https://user-images.githubusercontent.com/103123951/227842190-4e64fc9d-ffd1-4e9a-9aee-0f8936344260.png)

**Reply by rursache** [reply]
thanks, had the same issue and this fixed it for me. it should be automatically tho.
